polkadot-parachains/integritee-kusama/src/lib.rs

```diff
@@ -777,6 +787,37 @@ pub type EnsureRootOrAllTechnicalCommittee = EitherOfDiverse<
 	pallet_collective::EnsureProportionAtLeast<AccountId, TechnicalCommitteeInstance, 1, 1>,
 >;
 
+parameter_types! {
+	pub const HeartBeatTimeout: BlockNumber = 10;
+}
+
+impl pallet_porteer::Config for Runtime {
+	type RuntimeEvent = RuntimeEvent;
+	type WeightInfo = ();
+	type PorteerAdmin = EnsureRootOrMoreThanHalfTechnicalCommittee;
+	type HeartBeatTimeout = HeartBeatTimeout;
+	type TokenSenderLocationOrigin = EitherOfDiverse<
+		EnsureSignedBy<IntegriteePolkadotSovereignAccount, AccountId32>,
+		EnsureRoot<AccountId32>,
```

Comment:
this `EnsureRoot` could be the hook to fix broken bridging through governance on the receiver side, right?

Reply:
Yes, or to be used in benchmarking/testing code.